### PR TITLE
100% n.m.block

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/block/entity/vault/VaultClientData.mapping [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_gyznndxk net/minecraft/block/entity/vault/VaultClientData
+	FIELD f_afcpyqze previousSpin F
+	FIELD f_bflpzmfj ROTATION_SPEED F
+	FIELD f_scdptokm currentSpin F
+	METHOD m_galflnte currentSpin ()F
+	METHOD m_jzjpevbj updateSpin ()V
+	METHOD m_lpfqwfru previousSpin ()F
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/vault/VaultClientData.mapping]:
```suggestion
	METHOD m_galflnte getCurrentSpin ()F
	METHOD m_jzjpevbj updateSpin ()V
	METHOD m_lpfqwfru getPreviousSpin ()F
```

File: mappings/net/minecraft/block/entity/vault/VaultServerData.mapping [hunk 2]
```diff
@@ -0,0 +1,41 @@
+CLASS net/minecraft/unmapped/C_nsbycoiv net/minecraft/block/entity/vault/VaultServerData
+	FIELD f_bzryaxno itemsToEject Ljava/util/List;
+	FIELD f_gdjgafzz lastFailedInsert J
+	FIELD f_pymjabqb stateUpdatingResumesAt J
+	FIELD f_sngaxqvz KEY Ljava/lang/String;
+	FIELD f_taqrjfhn rewardedPlayers Ljava/util/Set;
+	FIELD f_xkrqkjsi totalEjectionsNeeded I
+	FIELD f_ygjbyfsp MAX_REWARDED_PLAYERS I
+	FIELD f_zbsyqpyj isDirty Z
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/vault/VaultServerData.mapping]:
```suggestion
	FIELD f_zbsyqpyj dirty Z
```

File: mappings/net/minecraft/block/entity/vault/VaultServerData.mapping [hunk 3]
```diff
@@ -0,0 +1,41 @@
+CLASS net/minecraft/unmapped/C_nsbycoiv net/minecraft/block/entity/vault/VaultServerData
+	FIELD f_bzryaxno itemsToEject Ljava/util/List;
+	FIELD f_gdjgafzz lastFailedInsert J
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/vault/VaultServerData.mapping]:
```suggestion
	FIELD f_gdjgafzz lastFailedInsertTime J
```
or
```suggestion
	FIELD f_gdjgafzz lastFailedInsertAt J
```
if we want to follow mojang's convention from `"state_updating_resumes_at"`
(setter+getter need updating, too)

File: mappings/net/minecraft/world/logic/Orientation.mapping [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	METHOD m_ennnjbdo fromIndex (I)Lnet/minecraft/unmapped/C_ctatbkvy;
+		ARG 0 i
+	METHOD m_fgyxqbmz getIndex ()I
+	METHOD m_fqnuqovh random (Lnet/minecraft/unmapped/C_rlomrsco;)Lnet/minecraft/unmapped/C_ctatbkvy;
+	METHOD m_ifqqcsgt getVerticalNeighbors ()Ljava/util/List;
+	METHOD m_lnhidaat getBias ()Lnet/minecraft/unmapped/C_ctatbkvy$C_ugxzsetw;
+	METHOD m_mwludmnc generateIndex (Lnet/minecraft/unmapped/C_xpuuihxf;Lnet/minecraft/unmapped/C_xpuuihxf;Lnet/minecraft/unmapped/C_ctatbkvy$C_ugxzsetw;)I
+		ARG 0 up
+		ARG 1 front
+		ARG 2 bias
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/logic/Orientation.mapping:27]: at least it's upfront about it

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/entity/vault/VaultServerData.mapping (hunk 3 above)
Comment (OroArmor) [reply mappings/net/minecraft/block/entity/vault/VaultServerData.mapping]: I like time better than at. I wonder if we should change the other one too time as well

Comment (supersaiyansubtlety) [reply mappings/net/minecraft/block/entity/vault/VaultServerData.mapping]: They should probably match, yeah

File: mappings/net/minecraft/world/logic/ExperimentalRedstoneUtils.mapping [hunk 5]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/unmapped/C_psmtqbgv net/minecraft/world/logic/ExperimentalRedstoneUtils
```
Comment (eristhea) [inline mappings/net/minecraft/world/logic/ExperimentalRedstoneUtils.mapping]: should be `Util` singular

File: mappings/net/minecraft/world/logic/Orientation.mapping (hunk 4 above)
Comment (eristhea) [reply mappings/net/minecraft/world/logic/Orientation.mapping:27]: lmao love this

**Review by eristhea — CHANGES_REQUESTED** [review]
looks great, just one thing to add!

File: mappings/net/minecraft/world/logic/ExperimentalRedstoneUtils.mapping (hunk 5 above)
Comment (OroArmor) [reply mappings/net/minecraft/world/logic/ExperimentalRedstoneUtils.mapping]: :blob_waaaaa:

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]

**Pull request merged**